.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,30 @@
+name: Feature request
+title: "[Feature] "
```

Comment:
Please don’t include these in the title. They are only useful if you don’t have labels, which are vastly superior.